src/redux/actions/projects/updateProject.js

```diff
@@ -4,6 +4,7 @@ import moment from 'moment';
 import {
   PROJECTS_UPDATE,
 } from '../../actionTypes/projects';
+import saveProject from './saveProject';
```

Comment:
why are you not using the action types for project here?

Reply:
for saving the project? It doesn't cause any changes to the reducer, so I don't think we need to dispatch any action type.